website/src/segment.tsx

```diff
@@ -85,13 +89,21 @@ export const AnnotatedForm = (
     return null
   }
   const showAnything = p.viewMode > ViewMode.Story
+  let wordCSS = css.wordGroupSelection.unselected
+  if (p.wordPanelDetails.currContents?.source === p.segment.source) {
+    wordCSS = css.wordGroupSelection.selected
+  }
   if (showAnything) {
     const showSegments = p.viewMode >= ViewMode.Segmentation
     const translation = p.segment.englishGloss.join(", ")
 
     return (
-      <div className={css.wordGroup} id={`w${p.segment.index}`}>
-        <div className={css.syllabaryLayer} lang="chr">
+      <div className={wordCSS} id={`w${p.segment.index}`}>
+        <div
+          className={css.syllabaryLayer}
+          lang="chr"
+          onClick={() => p.wordPanelDetails.setCurrContents(p.segment)}
```

Comment:
As I play with it, I'd like users to be able to highlight the text of a word without changing their selection state. So, can we only select the word if you click on the information icon in the top-right corner of each word? That's how the designs are envisioned.

For this, you should also change the cursor style for the info icon to be `select`. And please add what currently shows as a tooltip when you hover on the info icon to the word details panel, below the multi-line translation. And then get rid of the tooltip.

Reply:
That makes sense, but I'd like to clarify what you mean about the cursor style for the info icon. As far as I can tell, select is not a single cursor style but a category where there are multiple. The most similar I think is `text` which shows the cursor as an I-beam. Even still, I don't quite get why we would use this, as it usually shows text selection, and the info icon is simply something we click on. 

Do you mean having this text selection icon for the whole word except the info icon, or some other cursor style?